lib/hanami/cli/commands/app/install.rb

```diff
@@ -6,6 +6,19 @@ module Hanami
   module CLI
     module Commands
       module App
+        # The `install` command exists to provide third parties a hook for their own installation
+        # behaviour to be run as part of `hanami new`.
+        #
+        # Third parties should register their install commands like so:
+        #
+        # ```ruby
+        # if Hanami::CLI.within_hanami_app?
+        #   Hanami::CLI.after "install", MyHanamiGem::CLI::Commands::Install
+        # end
+        # ````
```

Comment:
![Screenshot 2022-11-14 at 14 01 40](https://user-images.githubusercontent.com/119904/201668494-b95ad8f8-a0cd-49f1-bd2e-63d030c73a38.png)

This doesn't render well, even though the markup for YARD is set to Markdown. I think it silently ignores this setting if it cannot find any markdown gem present. I added redcarpet to the `Gemfile` and after that it starts to render correctly:

![Screenshot 2022-11-14 at 14 11 40](https://user-images.githubusercontent.com/119904/201668732-70611980-5bbf-4a4a-9c1e-f76931803212.png)